src/components/column/column.tsx

```diff
@@ -11,30 +11,72 @@ import TasksList from '../task-list/tasks-list';
 interface ColumnProps {
   columnId: string;
   title: string;
+  order: number;
 }
 
-const Column: React.FC<ColumnProps> = ({ columnId, title }) => {
+const Column: React.FC<ColumnProps> = ({ columnId, title, order }) => {
   const dispatch = useAppDispatch();
   const { boardId } = useParams();
 
+  const [columnTitle, setColumnTitle] = useState(title);
+
   const handleDeleteOk = () => {
     if (boardId) {
       dispatch(columnsDelete({ boardId: boardId, columnId: columnId }));
```

Comment:
```suggestion
      dispatch(columnsDelete({ boardId, columnId }));
```
если одинаковые - можно не дублировать